packages/gen-ai-hub/src/orchestration/orchestration-completion-post-request.test.ts

```diff
@@ -46,10 +46,33 @@ describe('constructCompletionPostRequest()', () => {
     expect(completionPostRequest).toEqual(expectedCompletionPostRequest);
   });
 
+  it('with model configuration and empty template', async () => {
+    genaihubCompletionParameters.prompt.template = [];
+    const expectedCompletionPostRequest: CompletionPostRequest = {
+      orchestration_config: {
+        module_configurations: {
+          templating_module_config: {
+            template: []
+          },
```

Comment:
Note: This results in a bad request in the orchestration service, I am not sure, if we should restrict this use case somehow in the client.

Reply:
I have asked Franzi if they could add a `minItems` in the spec for `template` as well.